appservice/src/main/java/nu/yona/server/messaging/rest/MessageController.java

```diff
@@ -415,17 +415,13 @@ private void embedBuddyUserIfAvailable(BuddyMessageEmbeddedUserDto buddyMessage)
 		{
 			buddyMessage.getSenderUser()
 					.ifPresent(user -> buddyMessage.setEmbeddedUser(curieProvider.getNamespacedRelFor(BuddyDto.USER_REL_NAME),
-							createResourceAssembler(user).toResource(user)));
+							createUserResource(user, buddyMessage)));
 		}
 
-		private UserResourceAssembler createResourceAssembler(UserDto user)
+		private UserResource createUserResource(UserDto user, BuddyMessageEmbeddedUserDto buddyMessage)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "buddyMessage". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://10.127.10.107:9000/coding_rules#rule_key=squid%3AS1172)

Reply:
Done